user/user_manager.rs

```diff
@@ -99,7 +99,10 @@ impl UserManager {
             },
         );
         match update_result {
-            Ok(Ok(())) => Ok(()),
+            Ok(Ok(())) => {
+                // TODO: invalidate cache
```

Comment:
implement

Reply:
This is an outdated todo, and has already been [implemented in `TypeDBService`](https://github.com/typedb/typedb/pull/7287/files?show-viewed-files=true&file-filters%5B%5D=#diff-86c12c24889b5c2ffc1b18bb1ff46d8c7a137d416d8dd24f14cda7a87180523cR239)